src/index.tsx

```diff
@@ -358,6 +360,43 @@ function deleteSelectedElements() {
   }
 }
 
+function save() {
+  if (elements && elements.length > 0) {
+    const items = [...elements];
+    for (const item of items) {
+      item.isSelected = false;
+    }
+
+    localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(items));
+  }
+
+  window.removeEventListener("beforeunload", onUnload);
+}
+
+function restore() {
+  const el = localStorage.getItem(LOCAL_STORAGE_KEY);
+
+  if (el) {
+    const items = JSON.parse(el);
+    for (let item of items) {
+      item = generateDraw(item, "#000000", "#ffffff");
```

Comment:
@Fausto95 I just realized but the stroke and fill colors should be a property of the elements, not a global one. We'll need to update this.

For example, right now you can create a black rectangle, change the color to red, copy paste the black rectangle which will turn red. This is unexpected.

Reply:
Yes, sure! 
I can work on this as soon as I get some free time

Reply:
Handled by https://github.com/excalidraw/excalibur/pull/65